examples/fcma/mvpa_voxel_selection.py

```diff
@@ -45,22 +45,33 @@
     raw_data = []
     # all MPI processes read the mask; the mask file is small
     mask_img = nib.load(mask_file)
-    mask = mask_img.get_data()
+    mask = mask_img.get_data().astype(np.bool)
     epoch_info = None
     if MPI.COMM_WORLD.Get_rank()==0:
         raw_data = io.read_activity_data(data_dir, extension)
         epoch_list = np.load(epoch_file)
         epoch_info = io.generate_epochs_info(epoch_list)
+        # if leaving some subject out, an example
+        #epoch_info = [x for x in epoch_info if x[1] != 0]
     num_subjs = int(sys.argv[5])
     # create a Searchlight object
-    sl = Searchlight(sl_rad=2)
+    sl = Searchlight(sl_rad=1)
     mvs = MVPAVoxelSelector(raw_data, mask, epoch_info, num_subjs, sl)
     # for cross validation, use SVM with precomputed kernel
     # no shrinking, set C=10
-    clf = svm.SVC(kernel='rbf', shrinking=False, C=10)
+    clf = svm.SVC(kernel='linear', shrinking=False, C=1)
```

Comment:
Change the C parameter in the comment as well. Better yet, remove it from the comment.